src/pages/myNotes/myNotes.jsx

```diff
@@ -0,0 +1,100 @@
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardContent from "@material-ui/core/CardContent";
+import CardActions from "@material-ui/core/CardActions";
+import CloseIcon from "@material-ui/icons/Close";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import CardHeader from "@material-ui/core/CardHeader";
+import PropTypes from "prop-types";
+
+const useStyles = makeStyles({
+  root: {
+    marginBottom: '12px',
+    textAlign: "right",
+  },
+  text: {
+    textAlign: "left",
+  },
+  meta: {
+    fontSize: 10,
+    textAlign: "right",
+  },
+  active: {
+    backgroundColor: "aliceblue",
+  },
+  btn: {
+    padding: "3px",
+  },
+});
+
+function MyNotes({ notes, activeNoteId, setActiveNoteId }) {
+  const classes = useStyles();
+
+  return (
+    <div className="notes-list">
+      {notes.map((note) => {
+        return (
```

Comment:
Do we really need to write `return` statement here? I think it will be better to use:
 `{notes.map((note) => (<Note ... />)}`